ci/scripts/start_instance.sh

```diff
@@ -78,6 +78,23 @@ INSTANCE_INFORMATION=$(gcloud compute --project=${PROJECT} instances create ${IN
   --format=json)
 CREATE_EXIT_STATUS=$?
 
```

Comment:
I’d suggest a single while true loop with a break for exit 0. This way we don’t duplicate the start instance call.